te/GET_AND_RUN_DOCKER_IMAGE.py

```diff
@@ -158,7 +158,7 @@ def __update_run_cmd(self):
             self.lgr.debug("Output of executing 'which netstat' out%s= err=%s" %(out,err))
             if(bool(err) or not(bool(out))):
                 sys.exit(EXIT_NO_NETSTAT_CMD)
-            (out, err) = self.__exec_cmd("netstat -ltn | awk '{print $4}' | grep '[0-9]' | cut -d ':' -f2  | uniq")
+            (out, err) = self.__exec_cmd("netstat -ltn | awk '{print $4}' | grep '[0-9]'| grep -o '[^:]*$' | sort | uniq")
```

Comment:
The above command figures out free ports to allocate for internal services. It had a bug while parsing services listening on IPv4 and v6. Fixing the parser